src/components/map/marker.vue

```diff
@@ -15,12 +15,13 @@
         />
       </svg>
       <img
-        class="marker__image"
-        :src="checkImage"
-        :alt="imageKey"
-      >
+class="marker__image"
+:src="checkImage"
+:alt="imageKey" />
     </div>
-    <MglPopup><div>{{ name }}</div></MglPopup>
+    <MglPopup
```

Comment:
the popup should be removed because clicking already navigates - we won't see this popup

Reply:
let's keep the popup for debugging? like if clicking doesn't actually navigate to the page at least we can see the popup

Reply:
Hmm I don't really see too much use but can keep if want, maybe add a `// TO REMOVE`` comment